symfony/plugins/orangehrmAttendancePlugin/config/routes.yaml

```
@@ -16,10 +16,10 @@ apiv2_attendance_validate_datetime:
   controller: OrangeHRM\Attendance\Controller\AttendanceMockController::validateDateTime
   methods: [ GET ]
 
-#apiv2_attendance_records:
-#  path: /api/v2/attendance/records/{id?}
-#  controller: OrangeHRM\Attendance\Controller\AttendanceMockController::sendAttendanceResponse
-#  methods: [ GET, POST, PUT ]
+apiv2_attendance_record:
+  path: /api/v2/attendance/record/{id?}
```

not sure url should be like
```suggestion
  path: /api/v2/attendance/my/record/{id?}
```